example-cluster/README.md

```diff
@@ -15,3 +15,12 @@ $ tox -e example-cluster
 $ cd /work
 $ ./example-cluster/start.sh
 ```
+
+# To test tronview, tronctl, etc
+First, start Tron in a container using the above steps. Then in a different terminal,
+you will attach to that running container. There you will be able to run `tronctl`,
+`tronview` and others against the Trond master in the example cluster.
+
+```
+$ sudo docker exec -it `docker ps | grep examplecluster_master | cut -d' ' -f1` /bin/bash
```

Comment:
sudo .tox/example-cluster/bin/docker-compose -f example-cluster/docker-compose.yml run master

Reply:
Thanks for commenting on this, this line felt really hacky to me and I was hoping someone would give a better way.

But, this line doesn't seem to do what i want. This spins up another examplecluster_master container, instead of attaching to the existing one

Reply:
`sudo .tox/example-cluster/bin/docker-compose -f example-cluster/docker-compose.yml attach master` ?

I'm sure there is some docker-compose incantation that does what you want.

Not a blocker.